host_core/lib/host_core/config_plan.ex

```diff
@@ -5,50 +5,80 @@ defmodule HostCore.ConfigPlan do
   @behaviour Vapor.Plan
 
   alias Vapor.Provider.{Dotenv, Env}
+  alias HostCore.FilesConfigProvider
 
   @prefix_var "WASMCLOUD_LATTICE_PREFIX"
-  @hostkey_var "WASMCLOUD_HOST_KEY"
-  @hostseed_var "WASMCLOUD_HOST_SEED"
   @default_prefix "default"
 
   @impl Vapor.Plan
   def config_plan do
-    {host_key, host_seed} = HostCore.WasmCloud.Native.generate_key(:server)
+    wash_config = Path.join(System.user_home(), "/.wash/host_config.json")
```

Comment:
Have we verified this canonicalizes correctly for Windows machines?

Reply:
The docs for all of these functions claim to be platform independent.